schedule/security/alp/fde_regen_key.yaml

```diff
@@ -0,0 +1,8 @@
+name: check_tpm2
```

Comment:
we could have all the test related with tpm in one test suite instead of creating one per each, we might not expand them more and would look more simple, anyway if one of them fail is quite critical, wdyt?

Reply:
good catch, fixed the name!

Yes, that's a good idea. I'm already refactoring some stuff!

Reply:
I wasn't referring the name, but randomly I pointed to that line, good you noticed the copy/paste.
I was referring that there is not need to have a test suite per each tpm test, we could group them in one single test suite, less yaml to handle also.